ios/src/TGMapViewController.mm

```diff
@@ -205,6 +205,32 @@ - (void)pickFeatureAt:(CGPoint)screenPosition
     });
 }
 
+- (void)pickMarkerAt:(CGPoint)screenPosition
+{
+    if (!self.map && !self.mapViewDelegate) { return; }
+
+    screenPosition.x *= self.contentScaleFactor;
+    screenPosition.y *= self.contentScaleFactor;
+
+    self.map->pickMarkerAt(screenPosition.x, screenPosition.y, [screenPosition, self](const Tangram::MarkerPickResult* markerPickResult) {
+        CGPoint position = CGPointMake(0.0, 0.0);
+
+        if (!markerPickResult) {
+            if ([self.mapViewDelegate respondsToSelector:@selector(mapView:didSelectMarker:atScreenPosition:)]) {
+                [self.mapViewDelegate mapView:self didSelectMarker:0 atScreenPosition:position];
```

Comment:
That's another solution, but I'm a little anxious to give to the user more implementation callback to override from the default interface (3 more in that case if we want to be consistent between selectable components). That can be easily forwarded that way in the SDK though.